conans/test/unittests/client/generators/scons_test.py

```diff
@@ -33,3 +33,22 @@ def variables_setup_test(self):
         self.assertIn('    "conan_version" : "None",', scons_lines)
         self.assertIn('    "MyPkg_version" : "0.1",', scons_lines)
         self.assertIn('    "MyPkg2_version" : "3.2.3",', scons_lines)
+
+    def system_frameworks_libs_test(self):
+        # https://github.com/conan-io/conan/issues/7301
+        conanfile = ConanFile(TestBufferConanOutput(), None)
+        conanfile.initialize(Settings({}), EnvValues())
+        cpp_info = CppInfo("MyPkg", "")
```

Comment:
Let's use a rootpath to check that the generator is taking the proper variable (framework_paths)

```suggestion
        cpp_info = CppInfo("MyPkg", "/rootpath")
```

Reply:
This makes the test a bit annoying, because in Windows it will use backslash (escaped with double backslash), so the test was a bit more dirty, but I will do it.